packages/config/config/devices/0x010f/templates/fibaro_template.json

```
@@ -1,4 +1,9 @@
 {
+	"default_metadata_for_fibaro": {
```

```suggestion
	"default_metadata": {
```
The `_for_fibaro` part seems unnecessary in a Fibaro config file.

---

Agree. Didn't know what to call this. But dropping the fibaro definitely makes it better.